src/main/java/com/craftingguide/exporter/extensions/craftingguide/ItemIconDumper.java

```diff
@@ -32,7 +35,6 @@ public ItemIconDumper() {
     public void dump(ModPackModel modPack, AsyncStep dumpAllItemsStep) {
         this.remainingMods = new LinkedList<ModModel>();
         for (ModModel mod : modPack.getAllMods()) {
-            if (!mod.isEnabled()) continue;
```

Comment:
I understand why you've done this, but there's a bit more involved in getting the data ready for the site than just dumping the icons, items, and recipes.  Take a look at some of the other mod extensions.  At a minimum, if the exporter dumps all the mods, it will need to somehow answer all the questions that those extensions answer (e.g., mod icon, item groups, etc.).

Reply:
some of those will be a harder to get because of forge not collecting them but i may be able to use curseforge to get an icon for the mods.

Reply:
What I've been doing is just picking a single item which is "representative" of that mod, and exporting that for the website.  As you've observed, mods often don't have icons / logos at all, and even when they do, they're often not suitable for the site.  I've found that just picking a single item works best.  That's what the `IconicBlock` attribute is for.